contracts/interchain-token-service/src/msg.rs

```diff
@@ -33,6 +34,13 @@ pub enum ExecuteMsg {
     /// The admin is allowed to remove the ITS address of a chain for emergencies.
     #[permission(Elevated)]
     DeregisterItsContract { chain: ChainNameRaw },
+
+    #[permission(Governance)]
+    SetChainConfig {
+        chain: ChainNameRaw,
+        max_uint: Uint256,
+        max_target_decimals: u8,
+    },
```

Comment:
can't this be part of `RegisterItsContract`? Why make it a separate execute msg?

Reply:
see below